docs/2.guide/2.directory-structure/1.middleware.md

```diff
@@ -57,15 +57,15 @@ Unlike navigation guards in [the vue-router docs](https://router.vuejs.org/guide
 ::
 
 ::alert{type=warning}
-We recommend using the helper functions above for performing redirects or stopping navigation. Other possible return values described in [the vue-router docs](https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards) may work but there may be breaking changes in future.
+We recommend using the helper functions above for performing redirects or stopping navigation. Other possible return values described in [the vue-router docs](https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards) may work, but there may be breaking changes in the future.
 ::
 
 ## What Order Middleware Runs In
 
 Middleware runs in the following order:
 
 1. Global Middleware
-2. Page defined middleware order (if there are multiple middleware declared with the array syntax)
+2. Page-defined middleware order (if there are multiple middlewares declared with the array syntax)
```

Comment:
Thoughts welcome! That _order_ word before opening bracket didn't really fit nicely.
```suggestion
2. Page-defined middleware (Multiple middleware declared with the array syntax run in the order in which they are declared.)
```